tests/image.js

```diff
@@ -12,11 +12,17 @@ import { setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-util
 import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view';
 import ModelRange from '@ckeditor/ckeditor5-engine/src/model/range';
 import global from '@ckeditor/ckeditor5-utils/src/dom/global';
+import env from '@ckeditor/ckeditor5-utils/src/env';
 
 describe( 'Image', () => {
+	const initialEnvEdge = env.isEdge;
+
 	let editorElement, model, view, editor, document, viewDocument;
 
 	beforeEach( () => {
+		// Most tests assume non-edge environment but we do not set `contenteditable=false` on Edge so stub `env.isEdge`.
+		env.isEdge = false;
```

Comment:
You can use sinon for this
```js
sinon.stub( env, 'isEdge' ).get( () => true );
```
and restore the stub in `afterEach`
```js
env.isEdge.restore()
```
Or you can use sandbox from core test utils https://github.com/ckeditor/ckeditor5-core/blob/master/tests/_utils/utils.js#L26-L34

Reply:
But it's a detail.